### PR TITLE
Added IP-cameras support (RTSP protocol), ReadMe fixed.

### PR DESCRIPTION
**Pull request by andreichekhonin**
IP cameras through RTSP protocol support added.

### CODE REVIEW

**Comment by andreichekhonin** [comment]
@IRDonch @Wovchena please take a look.

File: demos/common/samples/args_helper.hpp [hunk 1]
```diff
@@ -34,8 +34,10 @@
 inline void readInputFilesArguments(std::vector<std::string> &files, const std::string& arg) {
     struct stat sb;
     if (stat(arg.c_str(), &sb) != 0) {
-        slog::warn << "File " << arg << " cannot be opened!" << slog::endl;
-        return;
+        if (arg.find("rtsp:") == std::string::npos) {
```
Comment (IRDonch) [inline demos/common/samples/args_helper.hpp]: A better test here would be `!= 0`, since something like `foo/rtsp:bar` isn't a camera address.

Comment (andreychekhoninintel) [reply demos/common/samples/args_helper.hpp]: Fixed

Comment (andreichekhonin) [reply demos/common/samples/args_helper.hpp]: Fixed.

**Comment by IRDonch** [comment]
Jenkins: test this please

**Pull request merged**